code/datums/traits/good.dm

```diff
@@ -69,6 +70,28 @@
 
 
 
+/datum/quirk/musician
+	name = "Musician"
+	desc = "You can tune handheld musical instruments to play melodies that clear certain negative effects and soothe the soul."
+	value = 1
+	mob_trait = TRAIT_MUSICIAN
+	gain_text = "<span class='notice'>You know everything about musical instruments.</span>"
+	lose_text = "<span class='danger'>You forget how musical instruments work.</span>"
+
+/datum/quirk/musician/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+
+	var/obj/item/instrument/guitar/guitar = new(get_turf(H))
```

Comment:
why not pick a random instrument with typesof/subtypesof, since the tuner works with anything

Reply:
why should it be a subtype

Reply:
Subtype wasnt the right word I meant to say different variations of the same quirk that are mutually exclusive and dont clutter the list so much

Reply:
cant you make it so you select the instrument when you select the trait

Reply:
If cyberboss couldnt figure that out for prosthetic limbs i doubt i can 😉

Reply:
with an attitude like that you wont

Reply:
In on_spawn() have it prompt the user with a query asking which instrument they'd want, if any. This way, they can even opt not to spawn with an instrument.

Alternatively, you could do some magic editing with the traits panel gui to allow for option selection, hopefully generalized, and then make this part of that system. It'd really help a lot of people, and potentially cut down on quirk clutter.